src/components/stories/NeynarProfileCard.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Meta, StoryFn } from "@storybook/react";
+import { ProfileCard, ProfileCardProps } from "../NeynarProfileCard/components/ProfileCard";
+import { NeynarProfileCard, NeynarProfileCardProps } from "../NeynarProfileCard/index";
+
+const meta: Meta<typeof NeynarProfileCard> = {
+  title: "NeynarProfileCard",
+  component: NeynarProfileCard,
+};
+
+export default meta;
+
+const Template: StoryFn<ProfileCardProps> = (args) => <ProfileCard {...args} />;
+const TemplateWithUser: StoryFn<NeynarProfileCardProps> = ({ fid, viewerFid }) => (
+  <NeynarProfileCard fid={fid} viewerFid={viewerFid} />
+);
+
+export const Primary = Template.bind({});
+Primary.args = {
+  username: "rish",
+  displayName: "rish",
+  avatarImgUrl: "https://i.imgur.com/naZWL9n.gif",
+  bio: "building /neynar 🪐 | neynar.com | /rish",
+  followers: 127364,
+  following: 676,
+  hasPowerBadge: true,
+  isOwnProfile: true,
+  isFollowing: true,
+  onCast: () => {},
+};
+Primary.argTypes = {
+  fid: { table: { disable: true } },
+  viewerFid: { table: { disable: true } },
+  onCast: { table: { disable: true } },
+};
+
+export const WithUser = TemplateWithUser.bind({});
+WithUser.args = {
```

Comment:
WithUser is breaking right now, even after being logged in, can we use the user from the  Neynar Context, i.e. if we signed into using NeynarAuthButon that user should be filled into this

@manan19 Should the components in SDK remember the state of the logged-in user? Or do we want to give flexibility to the devs to add their own state or should we do both?

![image](https://github.com/neynarxyz/react/assets/37466979/18a8c189-8cce-45f9-baed-6cdd97082f32)

Reply:
This was caused due to `NEYNAR_API_URL` was the part of .env which should not be the case